assets/README.md

```diff
@@ -0,0 +1,153 @@
+   3. Remove all the font family variants from the [fonts.css file](https://github.com/Expensify/App/blob/main/assets/css/fonts.css).
+
+#### Storybook (Web)
+
+The font files used by the Storybook web app are stored inside `assets/fonts/web/` folder.
+
+To add or remove a font used in the storybook web app:
+
+1. Add or remove the desired font files inside `assets/fonts/web/` folder.
+2. If you are adding a new font family into the project:
+   1. Add all the new font family variants to the [fonts.css storybook file](https://github.com/Expensify/App/blob/main/.storybook/fonts.css), replacing `<new_font_family>` with the font family name and `<font-family-file>` with the file name.
+      ```css
+      @font-face {
+          font-family: <new_font_family>;
+          font-weight: 500;
+          font-style: normal;
+          src: url('../assets/fonts/web/<font-family-file>.woff2') format('woff2'),  url('../assets/fonts/web/<font-family-file>.woff') format('woff');
+      }
+      /* Add the other variants too. */
+      ```
+3. If you are removing a font family from the project:
+   1. Remove all the font family variants from the [fonts.css storybook file](https://github.com/Expensify/App/blob/main/.storybook/fonts.css).
+
+#### Expensify Help (Web)
+
+The fonts files used by the Expensify Help web app are located inside `docs/assets/fonts/` folder.
+
+To add or remove a font used in the Expensify Help web app:
+
+1. Add or remove the desired font files inside `docs/assets/fonts/` folder.
+2. If you are adding a new font family into the project:
+   1. Add all the new font family variants to the [_fonts.scss file](https://github.com/Expensify/App/blob/main/docs/_sass/_fonts.scss), replacing `<new_font_family>` with the font family name and `<font-family-file>` with the file name.
+      ```scss
+      @font-face {
+        font-family: <new_font_family>;
+        font-weight: 500;
+        font-style: normal;
+        src: url('/assets/fonts/<font-family-file>.woff2') format('woff2'), url('/assets/fonts/<font-family-file>.woff') format('woff');
+      }
+      /* Add the other variants too. */
+      ```
+3. If you are removing a font family from the project:
+   1. Remove all the font family variants from the [_fonts.scss file](https://github.com/Expensify/App/blob/main/docs/_sass/_fonts.scss).
+
+### Using the fonts
+
+#### Native / Web / Desktop
+
+To use your fonts in the app, just import the desired font from [FontUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/FontUtils/index.ts) and use in your style objects.
+
+You should use `FontUtils.fontFamily.platform.<font-family-key>` for the most use cases in the app because it will use the correspondent set of font families (with fallback fonts or not) according to the platform. `FontUtils.fontFamily.single.` and `FontUtils.fontFamily.multi.` should only be used when we want a specific set of font families independently of the platform.
```

Comment:
```suggestion
You should use `FontUtils.fontFamily.platform.<font-family-key>` for most use cases in the app because it will use the correspondent set of font families (with fallback fonts or not) according to the platform. `FontUtils.fontFamily.single.` and `FontUtils.fontFamily.multi.` should only be used when we want a specific set of font families independently of the platform.
```